packages/KSelect/KSelect.vue

```diff
@@ -85,6 +85,19 @@
             style="position: relative;"
             role="listbox"
           >
+            <button
+              v-if="isClearVisible"
+              :class="{ 'overlay-label-chevron': overlayLabel }"
```

Comment:
This class actually needs be on the `KIcon` rather than the `button` to be able to hit the styles we have defined below.

Reply:
The styles below is defined as:
```css
.clear-selection-icon {
  /* some rules */

  &.overlay-label-chevron {
    /* some rules */
  }
}
```

so `.overlay-label-chevron` and `.clear-selection-icon` should be on the same element. I also changed `overlay-label-chevron` to `overlay-label-clear` just to be more clear.